Source/WebKit/UIProcess/mac/WKImmediateActionController.mm

```diff
@@ -175,6 +175,12 @@ - (void)immediateActionRecognizerWillPrepare:(NSImmediateActionGestureRecognizer
     _state = WebKit::ImmediateActionState::Pending;
     immediateActionRecognizer.animationController = nil;
 
+    if (!_page->mainFrame())
+        return;
+
+    if (!_page->mainFrame()->frameID())
```

Comment:
This one's just an ID so I don't think you have to check it? Also, I don't think you'll have a non null frame without an ID anyway.

Reply:
I think a 0 `frameID` would fail at decoding time — but yeah, it does seem like this check is unnecessary though, since a non-null frame implies non-zero ID.